client/my-sites/themes/controller.jsx

```diff
@@ -97,18 +97,11 @@ export function fetchThemeFilters( context, next ) {
 		return next();
 	}
 
-	const filtersRequest = wpcom.req.get( '/theme-filters', {
-		apiVersion: '1.2',
-		locale: context.lang, // Note: undefined will be omitted by the query string builder.
-	} );
-
-	const timeout = new Promise( ( _, reject ) => {
-		setTimeout( () => {
-			reject( new Error( 'Theme filters request timed out' ) );
-		}, 500 );
-	} );
-
-	Promise.race( [ filtersRequest, timeout ] )
+	wpcom.req
+		.get( '/theme-filters', {
```

Comment:
@arthur791004 will this method make the timeout longer or it remove the timeout completely?

Reply:
It removes the timeout completely. If we feel it's slow, we can consider how to add the timeout back in the future

Reply:
Good point